airbyte-integrations/connectors/source-faker/source_faker/spec.json

```
@@ -4,12 +4,12 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Faker Source Spec",
     "type": "object",
-    "required": ["count"],
+    "required": [],
     "additionalProperties": true,
     "properties": {
       "count": {
         "title": "Count",
-        "description": "How many users should be generated in total.  This setting does not apply to the purchases or products stream.",
+        "description": "How many users should be generated in total. By default, one purchase per user will also be created. This setting does not apply to the products stream.",
```

huh! I thought you got 10 purchases for every 10 users (not evenly distributed).  If you make 1K users by default, you should also get 1K purchases, if the stream is enabled

---

Maybe this is true. I've only observed that the number of records of those two streams always matched. I'll double check the logic and will update the description as needed before merging.

Thanks!

---

Thanks, @evantahler, for providing the correct spec on how records are generated. Updated per your comment into the below revised description:

```suggestion
        "description": "How many users should be generated in total. The purchases table will be scaled to match, with 10 purchases created per 10 users. This setting does not apply to the products stream.",
```